linkup-cli/src/env_files.rs

```diff
@@ -216,6 +214,25 @@ mod test {
         assert_eq!(expected_content, file_content);
     }
 
+    #[test]
+    fn clear_env_file_no_new_line_after_linkup() {
+        let content = format!(
+            "{}\n{}\n{}\n\n{}\n{}",
+            "EXISTING_1=VALUE_1",
+            "EXISTING_2=VALUE_2",
+            "##### Linkup environment - DO NOT EDIT #####",
+            "SOURCE_1=VALUE_1",
+            "##### Linkup environment - DO NOT EDIT #####",
+        );
+        let env_file = TestFile::create(&content);
+
+        clear_env_file("service_1", &env_file.path).unwrap();
+
+        let file_content = fs::read_to_string(&env_file.path).unwrap();
+        let expected_content = format!("{}\n{}", "EXISTING_1=VALUE_1", "EXISTING_2=VALUE_2",);
+        assert_eq!(expected_content, file_content);
```

Comment:
I wonder if linkup should always add a newline at the end of the file if there isn't one?

Reply:
We can do that as well! Right now we are explicitly removing a last new line if there is one 🤔

Reply:
https://github.com/mentimeter/linkup/blob/e60a633faaf41cdabf667b87a35912aff835c99e/linkup-cli/src/env_files.rs#L84-L86

Reply:
🤦

Reply:
I think this was to fix the case where this happened:

```
\n
\n
\n
```

and the file kept growing

Reply:
Maybe we can remove any repeating trailing `\n` to ensure there is only one left by the end of it? 🤔

Reply:
Yeah that sounds like a solid solution

Reply:
What do you think of 34ebcfb?